HACKING.md

```diff
@@ -264,28 +264,20 @@ that, to repack everything, run these commands:
 $ cd initrd
 $ find . | cpio --create --quiet --format=newc --owner=0:0 | lz4 -l -7 > ../initrd.img
 $ cd -
-$ apt download systemd-boot-efi
-$ dpkg --fsys-tarfile systemd-boot-efi_*.deb |
-       tar xf - ./usr/lib/systemd/boot/efi/linuxx64.efi.stub
+$ apt install systemd-boot-efi systemd-ukify
 $ objcopy -O binary -j .linux pc-kernel/kernel.efi linux
-$ llvm-objcopy --add-section .linux=linux --set-section-flags .linux=readonly,data \
-          --add-section .initrd=initrd.img --set-section-flags .initrd=readonly,data \
-          usr/lib/systemd/boot/efi/linuxx64.efi.stub \
-          pc-kernel/kernel.efi
+$ /usr/lib/systemd/ukify build --linux linux \
+          --initrd initrd.img \
+          --output pc-kernel/kernel.efi
```

Comment:
Probably worth adding the line if getting the arm64 kernel on x86, which is:
```
/usr/lib/systemd/ukify build --linux linux --initrd initrd.img --efi-arch aa64 --stub <path>/linuxaa64.efi.stub --uname <kernel_version>
```

Reply:
I have added `--efi-arch aa64`. The uname should be scraped by ukify from the kernel.

But... Now I see that the stub is wrong. But it was already wrong. It did `apt download systemd-boot-efi`, which downloaded the x86-64 stub.

Reply:
I have removed it. The section is called: `Hacking generic x86 initrd`.

Reply:
`--uname` is needed if you cross-build, otherwise `ukify` produces an error:
```
# /usr/lib/systemd/ukify build --linux linux --initrd initrd.img --efi-arch aa64 --stub usr/lib/systemd/boot/efi/linuxaa64.efi.stub 
Kernel version not specified, starting autodetection 😖.
Real-Mode Kernel Header magic not found
+ readelf --notes linux
readelf: Error: Not an ELF file - it has the wrong magic bytes at the start
Found uname version: 6.8.0-20-generic
Wrote unsigned linux.unsigned.efi
```
Right, the section is about hacking x86, but probably it can be made generic or a new section can be added - anyway, that can be in a follow-up.